src/sdk/DocumentSDK.ts

```diff
@@ -11,12 +11,13 @@ import * as Utils from "../lib/Utils";
 function calculateDocumentCreateOptionsDefault(options?: DocumentCreateOptions) {
     const randomDocID = crypto.randomBytes(16).toString("hex");
     if (!options) {
-        return {documentID: randomDocID, documentName: "", accessList: {users: [], groups: []}};
+        return {documentID: randomDocID, documentName: "", grantToAuthor: true, accessList: {users: [], groups: []}};
     }
     return {
         //Use the user provided ID or generate a random ID if not provided
         documentID: options.documentID || randomDocID,
         documentName: options.documentName || "",
+        grantToAuthor: options.grantToAuthor !== false,
```

Comment:
Hello darkness my old friend...

Reply:
I had to stare at this line for about 45 seconds to make sure it did what I wanted...